denon_config.ts

```diff
@@ -15,6 +15,7 @@ export interface DenonConfig {
   execute: { [extension: string]: string[] };
   fmt: boolean;
   test: boolean;
+  upgrade: string;
```

Comment:
Shouldn't be part of the config just like `--help` isn't

Reply:
remove!